cli/create/codegen/generate_mode.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/apex/log"
+	"github.com/dave/jennifer/jen"
+)
+
+func generateFileModeFile(path string, filename string) error {
```

Comment:
I think that it would be great to add a comment that shows result file structure.